Design-Docs/tools/scs-compatible.yaml

```diff
@@ -0,0 +1,35 @@
+name: SCS Compatible
+url: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Design-Docs/tools/scs-compatible.yaml
+iaas:
+  versions:
+    - version: v2
+      stabilized_at: 2023-10-03
+      standards:
+        - name: Flavor naming
+          url: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Standards/SCS-0003-v2-flavor-naming.md
+          check_tool: ./flavor-names-openstack.py
```

Comment:
ditch the leading "./". URLs can be easily identified by a leading "http" string.
By removing the leading "./" you'll also add the ability to expand the tool to auto-execute the programs in the future even when they reside in a different path.

Reply:
same for the following check_tool parameters

Reply:
Well, `./subdirectory/tool` would have worked without having to change the code.

To beautify this somewhat, I added to the spec that all check_tool file names are relative to the scs-compliance-tool.py path, which simplifies calling it from somewhere else. (Assuming that the main check tool and the individual checkers remain in the same git repo, this will work well.)
Implemented in commit adc8fc3f985997dc051747b93da643b4b17aab64.

Reply:
> same for the following check_tool parameters

I don't understand this comment, sorry.
The standards[].check_tool_args: takes a string that is passed to the tool (split by whitespace)

Reply:
Fine for me. The second command related to line 14, line 25, line 29 as they all contain the "check_tool" key.